stage0/src/dice_mfg_self.rs

```diff
@@ -3,13 +3,14 @@
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 use crate::dice::SerialNumbers;
-use dice_crate::{CertData, DeviceIdSelfMfg, DiceMfg, Handoff};
+use crate::Handoff;
+use dice_crate::{CertData, DeviceIdSelfMfg, DiceMfg};
 use lpc55_pac::Peripherals;
 use salty::signature::Keypair;
 
 pub fn gen_mfg_artifacts(
     deviceid_keypair: &Keypair,
-    _peripherals: &Peripherals,
+    _: &Peripherals,
```

Comment:
NIT: Seem unrelated to this PR. This was a style issue?

Reply:
That was a due to a couple of changes. I removed that line as I didn't think it was necessary, but then the dice code wouldn't build. I didn't realize the function is essentially part of an implicit trait. When I added it back, I just never added the rest of the name I guess, as it seemed redundant.